proxies/sandbox/apiproxy/resources/jsc/ConnectionMetadata.SetRequestHeaders.js

```diff
@@ -48,8 +48,13 @@ const nrlPointers = {
     return;
   }
 
+  var odsCodeExtension = context.getVariable(
+    "request.header.NHSD-End-User-Organisation"
+  );
```

Comment:
are we able to test the behaviour of this when `request.header.NHSD-End-User-Organisation` isn't provided? I think we need a smoke test from this end to be sure that this works for both the case that it is and isn't provided

Reply:
or at least a postman request

Reply:
You might need to do some apigee error management here as im not sure this will actually throw an error unless you drop out of the file if its missing

Reply:
This gets defaulted to null having tested this inside Apigee itself.